### PR TITLE
Rebase indices

### PR DESCRIPTION
**Pull request by dholms**
This ended up being less than I thought at first:
- one missing index on repo_blob
- slightly adjust a rebase query to ensure we're making use of the index on repo_commit_block

### CODE REVIEW

File: packages/pds/src/sql-repo-storage.ts [hunk 1]
```diff
@@ -158,6 +158,7 @@ export class SqlRepoStorage extends RepoStorage {
           .selectFrom('repo_commit_block')
           .selectAll()
           .where('repo_commit_block.creator', '=', this.did)
+          .where('repo_commit_block.commit', '=', rebase.commit.toString())
```
Comment (devinivy) [inline packages/pds/src/sql-repo-storage.ts:161]: I like that this also makes the query more specific and safer.  It should be the same resultset with or without this condition, but if for some reason it's not, this is more specific and would delete fewer blocks 👍

**Review by devinivy — APPROVED** [review]
Relaxed 🌴

File: packages/pds/src/sql-repo-storage.ts (hunk 1 above)
Comment (dholms) [reply packages/pds/src/sql-repo-storage.ts:161]: no it would actually delete more blocks i believe - this is a clause in a WHERE NOT EXISTS

Comment (devinivy) [reply packages/pds/src/sql-repo-storage.ts:161]: Oh yup of course.  Still seems all good to me.

**Pull request merged**